src/style.css

```
@@ -0,0 +1,134 @@
+* {
+  box-sizing: border-box;
+  --main-color: #060b35;
+  --secondary-color: #616070;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body,
+p {
+  margin: 0;
+}
+
+.page .page__card {
+  width: 200px;
+}
+
+.card {
+  padding: 32px 16px 16px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  color: var(--secondary-color);
+  margin-bottom: 16px;
+}
+
+.card__reviev {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-end;
+  margin-bottom: 24px;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(./images/star.svg);
+}
+
+.card__stars .stars__star:nth-child(-n + 4) {
+  background-image: url(./images/star-active.svg);
+}
+
+.card__stars .stars__star:last-child {
+  margin-right: 0;
+}
```

Delete redundant selectors
```suggestion
.stars__star:nth-child(-n + 4) {
  background-image: url(./images/star-active.svg);
}

.stars__star:last-child {
  margin-right: 0;
}
```